<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt001</class>
 <widget class="QDialog" name="qt001">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>词云01</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_file">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select_file">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_read">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cloud">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>生成词云</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gv_pic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>581</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>631</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
